inspirehep/modules/theme/bundles.py

```diff
@@ -65,7 +66,8 @@
         "clipboard": "~1.5.8",
         "flightjs": "~1.5.0",
         "angular": "~1.4.9",
-        "angular-sanitize": "~1.4.9"
+        "angular-sanitize": "~1.4.9",
+        "angular-ui-bootstrap": "~1.2.4"
```

Comment:
I think I have missed adding this dependency. So when you rebase you can add it.